sdk/eventhubs/azure-messaging-eventhubs-stress/src/main/java/com/azure/messaging/eventhubs/stress/scenarios/EventProcessorWithOptions.java

```diff
@@ -35,86 +36,137 @@
  * <p>
  * Support options:
  * <pre>
- * --UPDATE_CHECKPOINT boolean
+ * --NEED_UPDATE_CHECKPOINT boolean
+ * --UPDATE_CHECKPOINT_TIMEOUT_IN_SECONDS int
  * --NEED_SEND_EVENT_HUB boolean
- * --RECEIVE_BATCH_SIZE  int
- * --RECEIVE_BATCH_TIMEOUT int
+ * --MAX_BATCH_SIZE  int
+ * --BATCH_TIMEOUT_IN_MILLIS int
  * </pre>
  */
 @Service("EventProcessorWithOptions")
 public class EventProcessorWithOptions extends EventHubsScenario {
```

Comment:
reviewer might want to know the outline of what changed in this scenario.

Reply:
Both `EventProcessorWithOptions` and `EventSenderBuffered` logic is refered to the latest update of [long running test](https://github.com/anuchandy/azure-messaging-long-running-test/tree/main/eventhubs-long-running/src/main/java/com/azure/longrunning/test/eventhubs). I just clean up code structure and expose some arguments as scenario options.

Reply:
For `EventProcessorWithOptions`, the updates are:
- move some options from `ScenarioOptions` to this class, as they only used by this class/scenario.
- track event when send batch event takes over 5 seconds
- rename some private logging functions